sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVFileFormat.scala

```diff
@@ -163,7 +163,7 @@ class CSVFileFormat extends TextBasedFileFormat with DataSourceRegister {
 
 }
 
-private[csv] class CsvOutputWriter(
+private[sql] class CsvOutputWriter(
```

Comment:
I believe `private[sql]` can be removed per [SPARK-16964](https://issues.apache.org/jira/browse/SPARK-16964)